doc/source/user/advancedoptions.rst

```diff
@@ -208,7 +208,7 @@ You can customize the output using the following options:
 
 - ``--worker-name``: show only batches from a specific worker
 - ``--max-results``: limit the number of results shown (default: 20)
-- ``--batch-state``: filter by batch state (e.g. ``RUNNING``, ``TERMINATED``, ``FAILED``)
+- ``--batch-state``: filter by batch state (i.e. ``SUBMITTED``, ``RUNNING`` or ``FINISHED``)
```

Comment:
In principle it is not really needed to specify the list of states, as the `-h` already provides all the options available.